.github/workflows/publish.yml

```diff
@@ -6,24 +6,30 @@ on:
       - main
 
 jobs:
-  deploy:
+  main_build_test:
     runs-on: ubuntu-latest
     strategy:
       matrix:
         node-version: [10.16.x]
-
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@@v2
```

Comment:
you've got some double `@` for the checkout version, but as an aside I had a _bunch_ of issues with checkout v2, hence publishing uses v1 right now.

Reply:
Ah.. I saw v2 in pull_request workflow, so wondered if we just missed it here. Either way, reverting this change in favor of your previous suggestion.